scripts/apps/authoring/authoring/directives/WordCount.js

```diff
@@ -7,8 +7,8 @@ export function WordCount(gettextCatalog) {
             item: '=',
             html: '@'
         },
-        template: '<span class="char-count words">{{numWords}} <span translate>'
-            + gettextCatalog.getString('words') + '</span></span>',
+        template: '<span class="char-count words"><span translate>' +
```

Comment:
here it's also either `gettextCatalog` or `<span translate>`

Reply:
yes it was there before in WordCount and CharacterCount, so I've kept it. I'm not an expert in Angular, so I was not sure if `translate` was parsed if in `template:`. If so, it's useless indeed.

I'll change it thanks.